### PR TITLE
Capitalize words based on GNOME HIG

### PR DESCRIPTION
**Pull request by nana-4**
I was uncertain whether some headings should be capitalized (e.g. "Can't be edited in GUI"), so I didn't touch them.

#191

### CODE REVIEW

File: plugins/import_pil/oomox_plugin.py [hunk 1]
```diff
@@ -220,14 +220,14 @@ class Plugin(OomoxImportPlugin):
...
-            'display_name': _('Palette style'),
+            'display_name': _('Palette Style'),
             'reload_theme': True,
         },
         {
             'key': '_PIL_PALETTE_STRICT',
             'type': 'bool',
             'fallback_value': False,
-            'display_name': _('Stronger follow palette template'),
+            'display_name': _('Stronger Follow Palette Template'),
```
Comment (actionless) [inline plugins/import_pil/oomox_plugin.py]: i'm not sure about this one

File: oomox_gui/theme_model.py [hunk 2]
```diff
@@ -324,7 +324,7 @@ def merge_theme_model_with_base(whole_theme_model, base_theme_model, plugin_mode
         'key': 'TERMINAL_THEME_EXTEND_PALETTE',
         'type': 'bool',
         'fallback_value': False,
-        'display_name': _('Extend palette with more lighter/darker colors'),
+        'display_name': _('Extend Palette with More Lighter/Darker Colors'),
```
Comment (actionless) [inline oomox_gui/theme_model.py]: and this one

It Just Looks Too Unusual When I See Quite a Long Sentence Written All Capitalized

File: plugins/import_pil/oomox_plugin.py [hunk 3]
```diff
@@ -185,7 +185,7 @@ class Plugin(OomoxImportPlugin):
 
     theme_model_import = [
         {
-            'display_name': _('Import colors from image: '),
+            'display_name': _('Import Colors from Image:'),
```
Comment (actionless) [inline plugins/import_pil/oomox_plugin.py]: Also should we live the colon at the end of the headline? HIG is not saying anything on this regard

File: plugins/theme_arc/oomox_plugin.py [hunk 4]
```diff
@@ -56,7 +56,7 @@ def __init__(self, transient_for, colorscheme, theme_name, **kwargs):
                 },
                 OPTION_EXPORT_GNOME_SHELL_THEME: {
                     'default': False,
-                    'display_name': _("Generate theme for Gnome _Shell"),
+                    'display_name': _("Generate theme for GNOME _Shell"),
```
Comment (actionless) [inline plugins/theme_arc/oomox_plugin.py]: Theme?

File: oomox_gui/theme_model.py (hunk 2 above)
Comment (actionless) [reply oomox_gui/theme_model.py]: mb just the sentence itself should be rewritten to be shorter

File: plugins/theme_arc/oomox_plugin.py (hunk 4 above)
Comment (nana-4) [reply plugins/theme_arc/oomox_plugin.py]: According to HIG, "Sentence capitalization" should be used for the checkbox labels.

File: plugins/import_pil/oomox_plugin.py (hunk 3 above)
Comment (nana-4) [reply plugins/import_pil/oomox_plugin.py]: +1 for removing the colon at the end of the headline, but I didn't do that for this PR as I thought it is another issue.

File: plugins/theme_arc/oomox_plugin.py (hunk 4 above)
Comment (actionless) [reply plugins/theme_arc/oomox_plugin.py]: ah, i didn't realized it's not a theme, but export option :-)

**Comment by actionless** [comment]
thanks for this!

**Pull request merged**